src/main/webapp/user.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Get the user @param user from the user servlet.
+ * @param {String} user
```

Comment:
nit
```suggestion
 * @param {string} user
```
https://jsdoc.app/tags-type.html

Reply:
Interesting, I thought String was a custom class like in Java haha

Reply:
```suggestion
 * @param {string} user
```